data/boarding missions.txt

```diff
@@ -34,6 +146,106 @@ mission "Assisting Hai"
 	on offer
 		payment 30000
 		conversation
-			`Knowing you risked your life to save the <origin>, the captain gives you <payment>.`
+			`Knowing that you endangered your life to save the <origin>, the captain grants you <payment>.`
 				decline
 	destination Hai-home
+	
+mission "Assisting Unfettered Hai (Thief)"
+	assisting
+	repeat
+	to offer
+		random < 12
+	source
+		government "Hai (Unfettered)"
+	on offer
+		payment -300000
```

Comment:
What I said before still stands, and what I said about the pirate jobs applies here too.